controller/app/models/cartridge_type.rb

```diff
@@ -111,7 +111,10 @@ def self.cartridge_attributes(source, url=nil, text=nil)
   def activate
     self.priority = Time.now
     if self.save
-      self.class.where(name: name, :priority.lt => self.priority).set(:priority, nil)
+      self.reload
```

Comment:
Without reloading the object, `self.priority` would not be equal to `self.class.find(self.id).priority`. I'm not sure if this is the best way to resolve that problem and would appreciate any input.